build.sbt

```diff
@@ -23,11 +23,10 @@ val avroVersion = "1.9.2"
 val bigqueryVersion = "v2-rev20191211-1.30.3"
 val catsVersion = "2.0.0"
 val datastoreVersion = "1.6.3"
-val grpcApiVersion = "1.17.0"
 val guavaVersion = "28.2-jre"
 val jacksonVersion = "2.10.2"
 val jodaTimeVersion = "2.10.5"
-val protobufVersion = Option(sys.props("protobuf.version")).getOrElse("3.11.3")
```

Comment:
Keeping this simple for now since we're not trying to cover a cube of protoc/proto2/proto3 versions. We can keep this latest and `protoc-jar` to whatever available since it's usually a few versions behind.